source/run-command/run-command.rst

```diff
@@ -223,7 +223,7 @@ Drivers MAY expose a runCursorCommand API with the following syntax.
 
       /**
        * See the `cursorType` enum defined in the crud specification.
-       * @see https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read
+       * @see ../crud/crud.md#read
```

Comment:
Since this is in code should this still be a full link? (change-streams.md still uses a full link)

Reply:
Fixed